fortuna/model/mlp.py

```diff
@@ -55,29 +98,43 @@ class MLPDeepFeatureExtractorSubNet(nn.Module):
         The number of units of each hidden layer.
     activations: Tuple[Callable[[Array], Array]]
         The activation functions after each hidden layer. 
+    dense: ModuleDef
+        Dense module.
+    dropout: ModuleDef
+        Dropout module.
+    dropout_rate: float
+        Dropout rate.
     """
 
     @nn.compact
-    def __call__(self, x: Array, **kwargs) -> jnp.ndarray:
+    def __call__(self, x: Array, train: bool = False, **kwargs) -> jnp.ndarray:
```

Comment:
Here the return type signature is a `jnp.ndarray` but I think it should be a `jax.Array`